src/main/java/net/uaznia/lukanus/hudson/plugins/gitparameter/GitParameterDefinition.java

```diff
@@ -155,12 +161,16 @@ public void setSortMode(SortMode sortMode) {
 		this.sortMode = sortMode;
 	}
 
-	public String getTagFilter() {
-		return this.tagFilter;
+	public String getFilter() {
```

Comment:
I do not think it is safe to remove this public method. If other plugins have a dependency on this plugin, then all plugins that depend on this would need to be updated.